idl/Microsoft.UI.Xaml.idl

```diff
@@ -277,10 +277,20 @@ namespace MU_XC_NAMESPACE
     [WUXC_VERSION_MUXONLY]
     [webhosthidden]
     [default_interface]
+    [MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+    [MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
     runtimeclass XamlControlsResources : Windows.UI.Xaml.ResourceDictionary
     {
         XamlControlsResources();
         static void EnsureRevealLights(Windows.UI.Xaml.UIElement element);
+
+        [WUXC_VERSION_PREVIEW]
+        {
+            [MUX_DEFAULT_VALUE("false")]
+            Boolean UseCompactDictionary{ get; set; };
```

Comment:
Saying "Dictionary" here implies an implementation to me. Instead maybe "UseCompactDensity", "UseCompactLayout"? I don't even know if you need the "Use" prefix. XamlControlResources.CompactLayout?
@LucasHaines probably has some existing thoughts on name options for this.

Reply:
Maybe something like "CompactResources", it only applies to control resources and isn't really a dictionary or layout mechanism.

Reply:
UseCompactResources